front/app/containers/Admin/projects/project/analysis/TopBar/index.tsx

```diff
@@ -161,19 +160,15 @@ const TopBar = () => {
           />
         </Box>
         <Tasks />
-        {window.Intercom && (
-          <IconButton
-            iconName="info-solid"
-            a11y_buttonActionMessage={formatMessage(
-              translations.supportArticle
-            )}
-            onClick={() => {
-              window.Intercom('showArticle', 8316692);
-            }}
-            iconColor={colors.grey800}
-            iconColorOnHover={colors.black}
-          />
-        )}
+
+        <Button
+          icon="info-solid"
+          buttonStyle="text"
+          openLinkInNewTab
+          linkTo={formatMessage(translations.supportArticleLink)}
+          iconColor={colors.grey800}
+        />
```

Comment:
We switched from the component library Button so we could use the `linkTo`/`openLinkInNewTab` attributes right? I wonder if it would be good to add those `openLinkInNewTab` and `linkTo` attributes to our component library Button :thinking:

Reply:
Yes, exactly. The issue with doing that in the component library is that when it's an internal link, it depends on the react router so the component cannot be fully decoupled from the app as a component library component should be. But ideally, we would have something like `ButtonLink` instead of a duplicate `Button` component.